protocol.go

```diff
@@ -80,6 +80,7 @@ var clientProtocolInfo ProtocolInfo = ProtocolInfo{
 	Features: []ProtocolFeature{
 		StreamsFeature,
 		TransactionsFeature,
+		WatchersFeature,
```

Comment:
I think it's worth to update description like here: https://github.com/tarantool/go-tarantool/blob/ae90ad5557320909a7458e5def13bea40770243d/protocol.go#L79-L81

Reply:
Added.